test/MixedLanguage/cuda-1.cpp

```diff
@@ -8,7 +8,7 @@
 
 // REQUIRES: nvcc
 
-// RUN: (nvcc -c -std=c++17 %p/cuda-1.cu -o %t.o && \
+// RUN: (nvcc -c -Xcompiler -fPIC -std=c++17 %p/cuda-1.cu -o %t.o && \
```

Comment:
Should this use `%cpp_std` for both compilers? We'll eventually want to use a recent CUDA toolkit, no?